Production/crab/configs/2018/DY.txt

```diff
@@ -0,0 +1,10 @@
+sampleType=MC_18
+
+DYJetsToLL_M-50-amcatnloFXFX	    /DYJetsToLL_M-50_TuneCP5_13TeV-amcatnloFXFX-pythia8/RunIIAutumn18MiniAOD-102X_upgrade2018_realistic_v15-v1/MINIAODSIM
+DYJetsToLL_M-50-amcatnloFXFX_ext2	/DYJetsToLL_M-50_TuneCP5_13TeV-amcatnloFXFX-pythia8/RunIIAutumn18MiniAOD-102X_upgrade2018_realistic_v15_ext2-v1/MINIAODSIM
```

Comment:
SUBMITFAILED
with error:
Block /DYJetsToLL_M-50_TuneCP5_13TeV-amcatnloFXFX-pythia8/RunIIAutumn18MiniAOD-102X_upgrade2018_realistic_v15_ext2-v1/MINIAODSIM#29e36df7-ad7d-4ecb-82f4-3f54bc0e80bd contains more than 100000 lumis.

Reply:
It can happen if dataset is too large. In such cases a task can’t be submitted with an automatic splitting. FileBased splitting should be used instead and number of files per job should be estimated by hands. Maybe it will be better to split this big dataset into a separate file to simply the submission.